src/Spago/Prelude.hs

```diff
@@ -158,25 +160,36 @@ type Spago m =
   , MonadMask m
   )
 
-echo :: MonadIO m => Text -> m ()
-echo = Turtle.printf (Turtle.s Turtle.% "\n")
+output :: MonadIO m => Text -> m ()
+output = Turtle.printf (Turtle.s Turtle.% "\n")
 
-echoStr :: MonadIO m => String -> m ()
-echoStr = echo . Text.pack
+outputStr :: MonadIO m => String -> m ()
+outputStr = output . Text.pack
 
-tshow :: Show a => a -> Text
-tshow = Text.pack . show
+logStderr :: MonadIO m => String -> m () 
+logStderr = liftIO . System.IO.hPutStrLn System.IO.stderr
+
+logText :: MonadIO m => Text -> m ()
+logText = logStderr . Text.unpack
 
-echoDebug :: Spago m => Text -> m ()
-echoDebug str = do
+logDebug :: Spago m => Text -> m ()
+logDebug str = do
   hasDebug <- asks globalDebug
-  Turtle.when hasDebug $ do
-    echo str
+  when hasDebug $ do
+    logText $ str
+
+logWarning :: MonadIO m => Text -> m ()
+logWarning = logText . ("WARNING: " <>)
+
+logError :: MonadIO m => Text -> m ()
+logError = logText . ("Error: " <>)
```

Comment:
ok,  I'll just use my judgement for what should be 'application output' and what is 'application logging' and let you review after.   I'll try and get that done in the next couple days.

Reply:
@Benjmhart actually let's just go with renaming `output`/`outputStr` to `logInfo`/`logInfoStr` in this PR, we can always take the transfer to stderr in a later PR

The reason for this is that I'd like to merge as soon as possible, as basically any other changes will conflict with this (since we're renaming the logging function which is used everywhere)